src/main/java/org/spongepowered/common/event/tracking/PhaseTracker.java

```diff
@@ -422,6 +425,18 @@ public void printExceptionFromPhase(Throwable e, PhaseContext<?> context) {
         }
     }
 
+    private void checkPhaseContextProcessed(IPhaseState<?> state, PhaseContext<?> context) {
+        if (context.notAllCapturesProcessed()) {
```

Comment:
Make sure to make this toggle able for verbosity. We don't want to spam server owners if we missed some captures.